### PR TITLE
Fix transaction details icon and label

### PR DESCRIPTION
**Pull request by akalin-keybase**
Label was wrong for account-to-account transfers.

### CODE REVIEW

File: shared/wallets/transaction-details/container.js [hunk 1]
```diff
@@ -78,7 +78,6 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     title: 'Transaction details',
     transactionID: tx.txID,
     you: stateProps.you,
-    yourAccountName: tx.sourceType === 'ownaccount' ? tx.source : '',
```
Comment (akalin-keybase) [inline shared/wallets/transaction-details/container.js]: bug here -- should be tx.target sometimes (see above)

File: shared/wallets/transaction-details/index.js [hunk 2]
```diff
@@ -45,6 +45,36 @@ export type Props =
...
+        <Kb.Avatar
+          onClick={() => props.onShowProfile(props.counterparty)}
+          username={props.counterparty}
+          size={size}
+        />
+      )
+    case 'stellarPublicKey':
+      return <Kb.Icon type="icon-placeholder-secret-user-32" style={{height: size, width: size}} />
+    case 'otherAccount':
+      return <Kb.Icon type="icon-wallet-32" style={{height: size, width: size}} />
```
Comment (akalin-keybase) [inline shared/wallets/transaction-details/index.js]: right icon here

File: shared/wallets/transaction-details/index.js [hunk 3]
```diff
@@ -45,6 +45,36 @@ export type Props =
...
+    case 'keybaseUser':
+      return (
+        <Kb.Avatar
+          onClick={() => props.onShowProfile(props.counterparty)}
+          username={props.counterparty}
+          size={size}
+        />
+      )
+    case 'stellarPublicKey':
+      return <Kb.Icon type="icon-placeholder-secret-user-32" style={{height: size, width: size}} />
```
Comment (akalin-keybase) [inline shared/wallets/transaction-details/index.js]: right size here

**Comment by akalin-keybase** [comment]
cc @cecileboucheron does this look right?

screen shot 2018-10-24 at 8 49 31 pm
screen shot 2018-10-24 at 8 49 22 pm
screen shot 2018-10-24 at 8 49 27 pm

**Comment by cecileboucheron** [comment]
@akalin-keybase 
Icons look good now, thanks! 
2 things to be fixed in the Sender and Receiver sections: 
* the account names should be BodySemiboldItalic, right now they're BodySmallSemiboldItalic.
* the Stellar address keys should be BodySemibold. Right now they look like BodySmallSemibold.

For both changes, it shouldn't affect how they look like in the tsx row (split the code?).

**Comment by akalin-keybase** [comment]
@cecileboucheron thanks! I'll fix those things in a follow-up PR.

**Pull request merged**